CHANGELOG.md

```diff
@@ -15,6 +15,40 @@ From version 8.0.0 onwards, [Semantic Versioning](http://semver.org/) is used.
 _Nothing yet._
 
 
+## [8.1.0] - 2017-12-17
+
+See all related issues and PRs in the [8.1.0 milestone].
+
+### Added
+- :star2: New `MagicAutoloadDeprecation` sniff to detect deprecated __autoload functions as deprecated in PHP 7.2. [#540](https://github.com/wimg/PHPCompatibility/pull/540)
+- :star2: New `DynamicAccessToStatic` sniff to detect dynamic access to static methods and properties, as well as class constants, prior to PHP 5.3. [#535](https://github.com/wimg/PHPCompatibility/pull/535)
+- :star2: New `NewConstants` and `RemovedConstants` sniffs to detect usage of new/removed PHP constants. [#526](https://github.com/wimg/PHPCompatibility/pull/525)
+- :star2: New `OptionalRequiredFunctionParameter` sniff to check for missing function call parameters which were required and only became optional in a later PHP version. [#524](https://github.com/wimg/PHPCompatibility/pull/524)
+- :star: `RemovedGlobalVariables` : detection of the use of $php_errormsg with track_errors. [#528](https://github.com/wimg/PHPCompatibility/pull/528)
+- :star: `NewFunctions` : detection of all new PHP 7.2 functions added. [#522](https://github.com/wimg/PHPCompatibility/pull/522)
```

Comment:
If they're not present, we shouldn't check for them, since PHPCompatibility would be giving a false positive. We should report it as a bug in PHP I guess ;-)

Reply:
Two things about this:
* When I created the PRs for this, PHP 7.2 was still in beta, so it may have been committed afterwards.
* It could, of course, also be that it had already been committed, but that my search skills were not good enough ;-)

I think we may need to re-check whether the functions are in PHP 7.2 or not and if they are, add them to the sniff. If they are not, verify that the PHP 7.2 changelog has been adjusted to not mention them.

Reply:
Ok, found the relevant commits in the mean time, so I think we need to add these functions to the list:
* https://github.com/php/php-src/commit/1b797f7ad3da798827806012abd3fd2641c1bc14
* https://github.com/php/php-src/commit/de65a2243f5e52ccafc69889ab0b64f4481c5358

Looks like they were added in 7.0.21/23, 7.1.7/9 and 7.2.0 though, so at which version to tag them ?

Refs:
* http://php.net/manual/en/function.oci-register-taf-callback.php
* http://php.net/manual/en/function.oci-unregister-taf-callback.php

Reply:
That's utterly confusing... why the heck did they do that ?
I guess we should put it on 7.0.21 ?

Reply:
See my reply in PR #545 - https://github.com/wimg/PHPCompatibility/pull/545#issuecomment-352482331

Reply:
Just realized:  we haven't added the new Sodium functions yet either. (Changelogs got updated since the original PRs), so we may want to do that too.